### DISCUSSION TITLE
y-axis inverted or logarithmic

### DISCUSSION THREAD

**Discussion by Photelegy** [dbuezas/lovelace-plotly-graph-card#42] [Q&A] [answered]
1. How can I invert the y-axis (useful for "showing" the relative height based of the air-pressure of smartphones)?
So that 940hPa would be on top and 970hPa would be on the bottom.
![image](https://user-images.githubusercontent.com/61714583/168327460-c99be190-8a8e-4527-b833-9b093beda3cb.png)

2. How can I make the y-axis logarithmic (useful for light-sensor [lux] or corona-cases)?
So that I can see the small changes better (without zooming) because of the logarithm.
![image](https://user-images.githubusercontent.com/61714583/168327351-847bc275-7e1b-46fe-ad93-e5dc0e9ef9f7.png)

**Comment by dbuezas** [comment, marked as answer]
# 1. Reverse axis:

```yaml
layout:
  yaxis:
    autorange: reversed

```

<img width="490" alt="image" src="https://user-images.githubusercontent.com/777196/168421158-07b18d5f-9bcb-420f-9f2d-c9d8f2b5aa4c.png">


# 2. Log Scale
```yaml
layout:
  yaxis:
    type: log
```

<img width="498" alt="image" src="https://user-images.githubusercontent.com/777196/168421120-b7fe682f-e1fd-4c16-b5e6-ada9231c02d0.png">

# 3. Pro tip ;)
Most of the time you can find the answer by googlling "plotly.js [your question]". Example:  "plotly.js log scale"

**Reply by Photelegy** [reply]
Thank you very much!
